test/blackbox-tests/test-cases/foreign-stubs/fs-mode-dependant/fsmd-exe.t/c_stubs.c

```diff
@@ -0,0 +1,14 @@
+#define CAML_NAME_SPACE
+#include "caml/mlvalues.h"
+
+#ifdef NATIVE_CODE
+CAMLprim value caml_b_or_n(value unit)          /* ML */
```

Comment:
what is `/* ML */` for? also you can factor it a bit like the following (but I'm fine with your version too)

```c
CAMLprim value caml_b_or_n(value unit)
{
#ifdef NATIVE_CODE
  return Val_int(1);
#else
  return Val_int(0);
#endif
}
```

Reply:
`CAMLprim` is not necessary IIRC